Best Practices + 2 broken links fixed

Alex-ssvlabs opened the pull request:
- Added Best Practices page
- Two broken links fixed:
  - `docs/operators/operator-onboarding/verified-operators.md` the Verified Operator list link
  - `docs/operators/operator-node/node-setup/enabling-dkg/start-dkg-node/README.md` link to Manual Configuration page

RaekwonIII commented on docs/operators/operator-node/node-setup/best-practices.md:
```diff
@@ -0,0 +1,224 @@
+- **CPU frequency is a major factor**. Fast cores with moderate-to-high core count are the recipe. Ensure each server can turbo boost to high frequency, yet avoid agressive overclocking.
+- **Disk and Redundancy:** NVMe SSDs are mandatory. Many use RAID1 mirroring or have failover nodes ready in case of disk failure. Recommended to use ext4 if unsure about ZFS's complexities. Improving read/write speed has one of the greatest impacts.
+- **RAM Considerations:** Usually RAM is not the limiting factor. Abundant RAM enables disk caching for the execution client database – reducing disk reads and writes. ECC memory is advisable to protect against bit flips in long-running servers. 
+- **Form Factor and Data Center:** Run on a mix of bare metal servers in colocation data centers and cloud VM instances. In either case, redundancy across hardware and geographic distribution is a best practice. 
```
"a mix of" sounds a bit misleading. It almost sounds like they should have SSV node on bare metal, CL on cloud, and EL somewhere else. Can you be more specific?

RaekwonIII commented on docs/operators/operator-node/node-setup/best-practices.md:
```diff
@@ -0,0 +1,224 @@
+- **Hardware Resources:** Adequate CPU, sufficient RAM, and fast, reliable storage are necessary.
+
+## **Major impact**
+More obvious, more impactful.
```
"more obvious, more impactful"...peanuts? stones?

I know it's kinda obvious, but make this a one-liner description...

RaekwonIII commented on docs/operators/operator-node/node-setup/best-practices.md:
```diff
@@ -0,0 +1,224 @@
+***
+
+## **Minor impact**
+Less obvious, less impactful.
```
Same comment as above

RaekwonIII commented on docs/operators/operator-node/node-setup/best-practices.md:
````diff
@@ -0,0 +1,224 @@
+  ```
+  echo 1024 | sudo tee /sys/block/nvme0n1/queue/nr_requests
+  ```
+  - [Disable `atime` feature](https://ethdocker.com/Usage/LinuxSecurity/#noatime) to bump up the performance and increase the lifetime of your SSD. 
````
Goddamn that's good research! I did not know about these 3, at all...

RaekwonIII commented on docs/operators/operator-node/node-setup/best-practices.md:
```diff
@@ -0,0 +1,224 @@
+**Considerations:**  
+- **Besu and Teku** were reported to be not a great fit together, with SSV specifically.
+- **Erigon (EL)** has the largest ledger in ELs, is heavy on RAM usage, and lower sync speed. For home setups with limited resources it can cause issues.
+- **Nimbus (CL)** performs well on a low validator count, however, post ~250 it won't be able to scale properly and you might have to switch clients.
```
"...with more than ~250..."

vaclav-ssvlabs commented on docs/operators/operator-node/node-setup/best-practices.md:
```diff
@@ -0,0 +1,224 @@
+
+### **For Home Setups**
+- **Hardware:** Usual setup has 4- to 8-core CPU (focus on single-thread performance) and 32GB RAM.
+- **Disk:** NVMe SSDs are strongly recommended. 2TB size should serve you until ~2026. You can have 4TB to be on the safe side. Stick to `ext4` filesystem, as it has proven more performant. 
```
ext4 fs is more performant than what other fs? Or is it a generalization?

vaclav-ssvlabs commented on docs/operators/operator-node/node-setup/best-practices.md:
```diff
@@ -0,0 +1,224 @@
+- **Disk and Redundancy:** NVMe SSDs are mandatory. Many use RAID1 mirroring or have failover nodes ready in case of disk failure. Recommended to use ext4 if unsure about ZFS's complexities. Improving read/write speed has one of the greatest impacts.
+- **RAM Considerations:** Usually RAM is not the limiting factor. Abundant RAM enables disk caching for the execution client database – reducing disk reads and writes. ECC memory is advisable to protect against bit flips in long-running servers. 
+- **Form Factor and Data Center:** If you have multiple EL+CL setups - run them on different bare metal servers, in colocation data centers, or cloud VM instances. Redundancy across hardware and geographic distribution is a best practice. 
+- **Firewalls and DDoS Protection:** One way to handle DDoS is to have a firewall or load balancer. It's advised to use your provider's DDoS protection solutions (e.g. CloudFlare). The goal is to not block or throttle legitimate p2p ports – maintaining full peer connectivity.
```
I am not sure about this line as for m,e it doesn't make much sense,  I would rather just  say something along the lines to make sure DDoS protection and Firewalls allow access to p2p ports to allow full peer connectivity

vaclav-ssvlabs commented on docs/operators/operator-node/node-setup/best-practices.md:
```diff
@@ -0,0 +1,224 @@
+**Critical Ports:**
+  - **SSV P2P:** Typically 12001 UDP and 13001 TCP (or as specified in your configuration)
+  - **Execution P2P:** Typically 30303 TCP and UDP (for Geth and Nethermind)
+  - **Execution RPC:** HTTP 8545 and WS 8546 (for Geth and Nethermind)
```
http and ws ports should not be open to public! they should only be acessible to SSV, exposing them to public i dangerous as someone else can use or overload them

RaekwonIII replied:
Fantastic point, 1000% agree

Alex-ssvlabs commented:
What was changed:
- Mentions of EL/CL clients combinations deleted completely
- Kernel changes are batched into one section, moved to the very bottom, added ⚠️ to not proceed if unsure
- Clarified SSV vs EL/CL hardware requirements
  - Removed disk specifications
  - Added link to EthStaker hardware guide to follow
- Small stylistic changes as per Taiga's comments above
